trollmoves/client.py

```diff
@@ -470,7 +472,7 @@ def make_uris(msg, destination, login=None):
     duri = urlparse(destination)
     scheme = duri.scheme or 'ssh'
     dest_hostname = duri.hostname or socket.gethostname()
-    if socket.gethostbyname(dest_hostname) in get_local_ips():
+    if scheme not in ('s3') and socket.gethostbyname(dest_hostname) in get_local_ips():
```

Comment:
I'm not really found of listing "s3" here, as we would need to add also "gcs" if we add it as a mover in the future. But maybe this is premature optimization, to do what you want with this comment.

Reply:
I kinda expected there would be similar cases so did `not in ('s3')` instead of `!= 's3'` :grin: .